internal/core/command/config.go

```diff
@@ -27,3 +30,44 @@ type ConfigurationStruct struct {
 type WritableInfo struct {
 	LogLevel string
 }
+
+func (c *ConfigurationStruct) UpdateFromRaw(rawConfig interface{}) bool {
```

Comment:
Add docblock comments to each of these methods -- prefer you duplicate the comments from other existing implementations (i.e. core-data) for consistency's sake.